web/e2e/specs/dashboard/dashboard.ts

```diff
@@ -28,7 +28,7 @@ export default function(qaProjectDir) {
       await expect(dashboardPage.jobCount().getText()).toBe('0');
     });
 
-    it('should check default documents after redeploy', async function () {
+    xit('should check default documents after redeploy', async function () {
```

Comment:
"xit" works??

Reply:
its to ignore those tests. There is no way to check from GUI if redeploy is done. And it was consistently failing and causing subsequent tests to fail. Hence commented it out.

Reply:
I recommend then either deleting the tests (ideal) or leave them as ignored and explain why they're ignored. That prevents someone from seeing them 6 months from now and wondering why they're ignored and if they matter or not. 

Generally, I only like to ignore a test if it's expected to be work in the very near term. Otherwise, an ignored test usually just adds confusion. Best to delete it.

Reply:
Deleted the tests.